cl/lib/elasticsearch_utils.py

```diff
@@ -466,6 +496,33 @@ def build_es_base_query(search_query: Search, cd: CleanData) -> Search:
                 fields,
                 cd.get("q", ""),
             )
+        case SEARCH_TYPES.PEOPLE:
+            child_query_fields = {
+                "position": add_fields_boosting(
+                    cd, ["appointer", "supervisor", "predecessor"]
```

Comment:
I think here we need to add all the position fields that were present in `person_text.txt` so those fields can be searchable by the text query, e.g: `position_type_display`, `nomination_process_display`, `organization_name`, `court.pk`, etc.

Reply:
you're right! the `person_text.txt` file includes a few fields from the `position` document.